debug_module.tex

```diff
@@ -242,18 +242,24 @@ \section{Program Buffer} \label{programbuffer}
 
 A debugger can write a small program to the Program Buffer, and then
 execute it exactly once with the Access Register Abstract Command,
-
 setting the \Fpostexec bit in \Rcommand.
-If \Fprogbufsize is 1, the Program Buffer may only hold a single instruction.
-This can be a 32-bit
+The debugger can write whatever program it likes (including jumps out of the
+Program Buffer), but the program must end with
+{\tt ebreak} or {\tt ebreak.c}. To save hardware, an implementation may support
+an implied {\tt ebreak} that is executed when a hart runs off the end of the
+Program Buffer. With this feature, a Program Buffer of just 2 32-bit words can
```

Comment:
shouldn't we say "as indicated by \Fimpebreak" or something similar?

Reply:
Good idea.